src/index.html

```diff
@@ -6,9 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img src="images/logo.png" alt="moyo__logo">
+      </div>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav__link is-active" >Apple</a></li>
+          <li><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li><a href="#smertphones" class="nav__link">Smartphones</a></li>
+          <li >
+            <a
+            href="#laptops&computers"
+            class="nav__link"
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li><a href="#tablets" class="nav__link">Tablets</a></li>
+          <li><a href="#photo" class="nav__link">Photo</a></li>
+          <li><a href="#video" class="nav__link">Video</a></li>
```

Comment:
please, format your html according to the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

Reply:
for ex:
```html
<li>
  <a href="#apple" class="nav__link is-active" >
    Apple
  </a>
</li>
```